packages/picasso-codemod/src/v22.0.0/remove-asterisks/__testfixtures__/basic.output.tsx

```diff
@@ -0,0 +1,28 @@
+import { Form, FormLabel } from '@toptal/picasso'
+import { Form as PicassoForm } from '@toptal/picasso-forms'
+// @ts-nocheck
+import React from 'react'
+
+const FORM_CONFIG_PROPS = {
+  foo: 'abr'
+}
+
+const requiredVariant = 'test'
+const required = true
+const label = 'Label'
+
+export default () => (
+  <>
+    <PicassoForm.ConfigProvider value={{}}>
```

Comment:
```suggestion
    <PicassoForm.ConfigProvider>
```
☝️ I would change this and add another variant when there are multiple `Form`-level configurations, so the rest of configs would remain